op-e2e/setup.go

```diff
@@ -337,13 +338,24 @@ func (cfg SystemConfig) start() (*System, error) {
 
 	// Configure connections to L1 and L2 for rollup nodes.
 	// TODO: refactor testing to use in-process rpc connections instead of websockets.
+
+	l1EndpointConfig := l1Node.WSEndpoint()
+	useHTTP := os.Getenv("OP_E2E_USE_HTTP") == "true"
+	if useHTTP {
```

Comment:
This is pretty cool! Would add maybe add a test log statement to indicate if http is being used or not.